setup.py

```diff
@@ -52,6 +52,7 @@ def load_requirements(*requirements_paths):
     version=VERSION,
     author='edX',
     description='An API for creating submissions and scores.',
+    long_description="An API for creating and scoring submissions for the open edX platform",
```

Comment:
Nit: This is the correct branding: 

```suggestion
    long_description="An API for creating and scoring submissions for the Open edX platform",
```